zio/csvio/ztests/error.yaml

```diff
@@ -0,0 +1,21 @@
+zql: '*'
+
+input: |
+  #0:record[error:error]
+  0:[-;]
+  0:[;]
+  0:[";]
+  0:[,;]
+  0:[\u002d;]
+  0:[\u005c;]
+
+output-flags: -f csv
+
+output: |
+  error
+  -
```

Comment:
I don't know that this has any meaning except maybe to a zeek person who knows zeek TSV.  I would be inclined to map nulls to csv empty string rather than minus.

Reply:
I guess the code was already written this way...

Reply:
This should indeed be an empty string, not "-", and is one of the problems identified by #1326. It'll be fixed in a subsequent PR.